lib/puppet/transaction/report.rb

```diff
@@ -70,6 +70,10 @@ class Puppet::Transaction::Report
   #
   attr_reader :metrics
 
+  # A hash of number of changes per resource
+  # @return [Hash<{String => Hash<{String => Event Status}>}] A map of type+title to number of events
```

Comment:
This comment doesn't seem to be correct, based on `sumup`.  This is actually a `Hash<String => Hash<String => Array<Status>>`, correct?